file_organizer_v2/src/file_organizer/services/audio/metadata_extractor.py

```
@@ -0,0 +1,353 @@
+        try:
+            from mutagen import File as MutagenFile
+            from mutagen.id3 import ID3
+            from mutagen.mp4 import MP4
+            from mutagen.flac import FLAC
+            from mutagen.oggvorbis import OggVorbis
+
+        except ImportError:
+            raise ImportError(
+                "mutagen is required for audio metadata extraction. "
+                "Install it with: pip install mutagen"
+            )
```

Ruff (0.14.13)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)